File: BugTracker.API/Models/Comment.cs
using System;

namespace BugTracker.API.Models
{
    public class Comment
    {
        public int Id { get; set; }
        public string Content { get; set; }
        public DateTime Created { get; set; }
        public DateTime Updated { get; set; }
        public bool IsDeteled { get; set; }
        public User Commenter { get; set; }
        public Ticket Ticket { get; set; }
    }
}

File: BugTracker-SPA/src/app/_models/Comment.ts
import { Time } from '@angular/common';
import { User } from './User';
import { Ticket } from './Ticket';

export interface Comment {
    id: number;
    content: string;
    created: Date;
    updated: Date;
    isDeleted: boolean;
    commenterId: string;
    userName: string;
    ticket: Ticket;
    ticketId: number;
}
